src/OpenLoco/Viewport.hpp

```diff
@@ -114,32 +114,76 @@ namespace OpenLoco::Ui
         int getRotation() const;
         void setRotation(int32_t value);
 
+        struct ScreenToViewport
```

Comment:
Any particular reason these are structs rather than namespaces?

Reply:
Can't nest a namespace within a struct/class. I'm not too sure if they should be part of this struct though.

Reply:
It seems a bit strange to me, without having any data members. Might be better to move them out?